README.md

```diff
@@ -49,164 +79,3 @@ SPANDX_CONFIG="./config/spandx.config.js" bash $PROXY_PATH/scripts/run.sh
   
 ## Versioning
 This project uses semantic versioning https://semver.org/. This process is automated by using [Semantic Release](https://github.com/semantic-release/semantic-release)
-
-## Patternfly
```

Comment:
I would say we need Patternfly and Insights Components in the documentation as they are only RedHat Insights related. Anyone coming from outer world would not have any idea about the fact that we are using patternly and other RedHat only dependencies (I would add more of these dependencies) . But, I totally agree with removing used technologies as they are common knowledge for any front-end developer.

Reply:
great point! I take it as granted :( . I'll it bring back